Yoast/yoast-components
# Removed logic for decoding seps

## Summary

This PR can be summarized in the following changelog entry:

* Removed logic for decoding seperators.

## Relevant technical choices:

* Removed it here, as it's yoast-specific. Moved it to wordpress-seo in https://github.com/Yoast/wordpress-seo/pull/9849

## Test instructions

This PR can be tested by following these steps:

* See sibling pr: https://github.com/Yoast/wordpress-seo/pull/9849

Fixes #550
